python/foglamp/services/core/api/browser.py

```diff
@@ -294,8 +294,18 @@ def validate_limit_skip(request, _dict):
     The amount of time covered by each returned value is set using the
     query parameter group. This may be set to seconds, minutes or hours
 
-    Return the result of the query
-    SELECT user_ts AVG((reading->>'reading')::float) FROM readings WHERE asset_code = 'asset_code' GROUP BY user_ts
+    Return the result of the query:
+
+    SELECT min((reading->>'reading')::float) AS "min",
+           max((reading->>'reading')::float) AS "max",
+           avg((reading->>'reading')::float) AS "average",
+           to_char(user_ts, 'YYYY-MM-DD HH24:MI:SS') AS "timestamp"
+    FROM foglamp.readings
+           WHERE asset_code = 'asset_code' AND
+             reading ? 'reading'
```

Comment:
If `AND` required then we should add into payload itself otherwise remove it from here.

Reply:
This is just the example query, taken from SQL seen by the storage

Reply:
The AND reading ? 'reading' is internal top the storage layer to check that the JSON property exists and should not be in the payload passed to the storage layer.